.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,68 @@
+name: trivy  ⏲
+
+env:
+  REGISTRY: ghcr.io
+
+on:
+  schedule:
+    - cron: '44 19 7,21 * *'
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        image:
+          - r: '4.2.0'
+            bioc: '3.15'
+            distro: rstudio
+    steps:
+      - name: Checkout repository 💳
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        id: buildx
+        with:
+          install: true
+
+      - name: Log in to the Container registry 🗝
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@7b7aa264d83dc58691451798b4d117d53d21edfe
+        with:
+          image-ref: 'ghcr.io/insightsengineering/rstudio_4.1.2_bioc_3.14:latest'
```

Comment:
Make this a matrix, to match the one in `scheduled.yaml`

Reply:
Using only one `rstudio-local`